src/core/packages/notifications/browser-internal/src/notification_coordinator.ts

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import * as Rx from 'rxjs';
```

Comment:
nit: Can we confirm that this global import does not affect our bundle size?

I'd expect `import type { Observable } from 'rxjs';` to be different to `import * as Rx from 'rxjs';` + refer to the type like `Rx.Observable`.

The reason is that `Observable` is also a valid class, which I expect to be loaded in the global import. I may be wrong, though.